### ISSUE TITLE
Slash commands require an extra 'return' keypress to send

### ISSUE THREAD

**Issue by dbkr** [vector-im/element-web#21860]
### Steps to reproduce

1. Focus composer
2. Type '/devtools'
3. Press return

#### What did you expect?
The devtools dialog

#### What happened instead?
The completion was dismissed and then nothing

Probably the most sensible thing to do would be for the completion to dismiss once you've typed, '/devtools' in full since you've already typed the whole thing so there's nothing left to autocomplete.

### Will you send logs?

No

**Comment by t3chguy** [comment]
This is the autocomplete taking the first Return to mean `accept completion`

**Comment by t3chguy** [comment]
This matches Discord & Slack and is for accessibility purposes
